woocommerce/woocommerce
# Show qty refunded on order views for customers

Fixes #23003

Shows the qty - refunded qty on customer facing order screens. Does not adjust totals however, as this could get confusing. The refunded $$ is shown below the items table.

To test,

1. Create an order with several line items
2. Refund some lines
3. View the my account page
4. View an order on the my account page
5. Send yourself an invoice email and check qtys

> Show qty refunded on customer facing order screens.
